src/server/utils/prepare.js

```diff
@@ -10,21 +10,43 @@ var exec = require('child_process').exec,
  * @return {Promise}
  */
 function execCordovaPrepare(projectRoot, platform) {
-    var deferred = Q.defer();
+    return retryAsync(getExecCordovaPrepareImpl(projectRoot, platform), 2);
```

Comment:
Sure. I didn't do anything like that because I was essentially just moving existing code. But no harm in making it clearer :). In fact I might move the `retryAsync` method to `jsUtils` and make 2 retries *it's* default.